tests/app/main/views/organisations/test_organisations.py

```diff
@@ -972,17 +971,55 @@ def test_organisation_settings_table_shows_letter_branding_pool(
     client_request.login(platform_admin_user)
     page = client_request.get(".organisation_settings", org_id=organisation_one["id"])
 
-    email_branding_options_row = page.select("tr")[10]
+    email_branding_options_row = page.select("tr")[9]
```

Comment:
oops, one other copy paste (even if this one is from a previous PR). be good to make this one use that find element function
```suggestion
    letter_branding_options_row = find_element_by_tag_and_partial_text(page, "tr", "Letter branding options")
```

Reply:
Thanks, this is now fixed too